packages/grafana-data/src/themes/palette.ts

```diff
@@ -1,5 +1,5 @@
 export const palette = {
-  white: '#fff',
+  white: '#FFFFFF',
```

Comment:
Seems like it should be unnecessary.

Reply:
The fusebit and the dark theme will work properly without this change, but the light theme will still throw the `Illegal value for token color: #fff` error.

Should i revert it anyways?